src/notation/view/notationpaintview.h

```diff
@@ -149,20 +155,19 @@ private slots:
     void onPlayingChanged();
     void movePlaybackCursor(uint32_t tick);
 
-    const Page* point2page(const QPointF& p) const;
+    const Page* pointToPage(const QPointF& point) const;
     QPointF alignToCurrentPageBorder(const QRectF& showRect, const QPointF& pos) const;
 
     QColor m_backgroundColor;
     notation::INotationPtr m_notation;
     QTransform m_matrix;
-    NotationViewInputController* m_inputController = nullptr;
-    PlaybackCursor* m_playbackCursor = nullptr;
-    NoteInputCursor* m_noteInputCursor = nullptr;
+    QScopedPointer<NotationViewInputController> m_inputController;
+    QScopedPointer<PlaybackCursor> m_playbackCursor;
+    QScopedPointer<NoteInputCursor> m_noteInputCursor;
```

Comment:
Maybe `std` is better? (for unification )

Reply:
replaced